conformity/fields/basic.py

```diff
@@ -191,6 +191,15 @@ class Float(Integer):
     introspect_type = "float"
 
 
+class Decimal(Integer):
```

Comment:
This, and `class float(Integer)`, feels strange. Should it be a different name `Integer` -> `Number`?

Reply:
Perhaps, but such a refactor seems outside the scope of this simple change.